src/lib/endpoints.js

```diff
@@ -0,0 +1,6 @@
+import { httpGet } from "./dataAccess";
+
+export const getAllScenes = async () => {
+    const response = await httpGet("/api/admin/v1/scenarios");
```

Comment:
Shouldn't this also deal with errors? If backend returns errors? Like cookie expired, or something went wrong?

Reply:
Currently we just throw an error as defined [here](https://github.com/uwblueprint/cdc-frontend/blob/1fa622173324b21e72411c82e9e58bbcdb8e8df0/src/lib/dataAccess.js#L5). I could add error handling but planning on doing it for another PR.